### ISSUE TITLE
Clipboard is not instantiable while building Bouncer, Guard

### ISSUE THREAD

**Issue by Axieum** [JosephSilber/bouncer#463]
Using Laravel `6.0.1` on version `v1.0.0-rc.6`, when running my PHPUnit tests I get this error wherever I use the Bouncer facade:
```log
Illuminate\Contracts\Container\BindingResolutionException : Target [Silber\Bouncer\Contracts\Clipboard] is not instantiable while building [Silber\Bouncer\Bouncer, Silber\Bouncer\Guard].
```

Here is the affected code:
```php
use Silber\Bouncer\BouncerFacade as Bouncer;
// ...
Bouncer::can('view.email', $this->resource);
```

I have tried the following to no avail:
* composer dump-autoload
* php artisan package:discover
* php artisan clear-compiled
* php artisan cache:clear
* php artisan migrate:fresh

I can see a few related issue posts, but they all seem to have been fixed, perhaps this is introduced in Laravel 6?

**Comment by Axieum** [comment]
It appears that registering the provider (and maybe alias) resolves this issue. Hence, Laravel automatic package discovery does not appear to be working in Laravel `6`.

**Comment by JosephSilber** [comment]
I tried to reproduce this issue, but I couldn't.

Here's a repo with a simple installation of Bouncer: https://github.com/JosephSilber/bouncer-issue-463-demo

**Comment by Axieum** [comment]
Okay, I have successfully ran the repository you supplied, and there are **no errors**. Here is my repository in question, and the error is raised upon running the PHPUnit tests: Axieum/laravel-api-boilerplate@7805ba00bf757dad2639331dc4943307aa14ad1b

I added a unit test to your repo, and it does not have the same error...

*I have tried re-installing the `/vendor` packages but to no avail.*

**Comment by JosephSilber** [comment]
There's way too much unrelated stuff going on in that repo.

Can you try to create a more isolated test case, which doesn't involve all of the other stuff in that repo?

**Comment by Axieum** [comment]
Okay, so I started a brand new project, setup a permission seeder and it all works. I went to add the `db:seed --class=PermissionSeeder` to the `tests/bootstrap.php` file, only to realise that somehow my PHPUnit installation is just... not right. Perhaps this was fixed in Laravel `6.0.1` as I made the project originally in `6.0.0`. I'm not sure if anything is messed up, so I'm going to close this issue and start from scratch with a `6.0.1` installation.

Reference: https://github.com/Axieum/laravel-api-boilerplate/commit/cfcb6b85a8bc1aa6f0448b06f84ed1ba1991a91e#diff-10eafd8bf5fbdde867b86fe38d30b9bb
